compose/install.md

```diff
@@ -4,37 +4,53 @@ keywords: compose, orchestration, install, installation, docker, documentation
 title: Install Docker Compose
 ---
 
-You can run Compose on macOS, Windows and 64-bit Linux. To install it, you'll need to install Docker first.
+You can run Compose on macOS, Windows and 64-bit Linux.
 
-To install Compose, do the following:
+## Prerequisites
 
-1.  Install Docker Engine:
+Docker Compose relies on Docker Engine for any meaningful work, so make sure you
+have Docker Engine installed either locally or remote, depending on your setup.
```

Comment:
@shin- Okay I'm not sure if this is well-worded to convey the right thing, but let me know what you think.